### PR TITLE
fix for generating compressed tarball

### PR DESCRIPTION
**Pull request by tenstormavi**
Fix for #536

### CODE REVIEW

**Comment by ndokos** [comment]
Is it worth compressing only to uncompress and unpack on the controller? Maybe yes, in which case, this is the right fix; but if not, the only thing needed is cleaning up confusing names (foo.tar.xz -> foo.tar) and maybe clean up comments.

**Comment by portante** [comment]
I think we want the data compressed.

**Comment by portante** [comment]
You are right, @ndokos, I think we don't want to actually compress the data, since we are going to unpack it anyways.  In some cases compresses will be nice, because it saves bandwidth, but most of these tar balls are small, so I don't think the gain is worth it at this point.

File: agent/util-scripts/pbench-remote-sysinfo-dump [hunk 1]
```diff
@@ -28,7 +28,7 @@ fi
 
 cd $sysinfo_path >&2
 echo "pbench-sysinfo-dump data size: $(du -sm $sysinfo_path | awk '{print $1}') MB" >&2
-tar cf - *
+tar --xz -cf - *
```
Comment (portante) [inline agent/util-scripts/pbench-remote-sysinfo-dump]: Let's revert this change, and instead, in `pbench-collect-sysinfo`, just drop the `.xz` suffix from the local file name.

**Comment by tenstormavi** [comment]
@portante @ndokos changes made as requested.

**Comment by tenstormavi** [comment]
@portante @ndokos changes made as requested.
